src/components/main-view/main-view.jsx

```diff
@@ -1,59 +1,121 @@
-import { useState } from "react";
-import {MovieCard} from "../movie-card/movie-card";
-import {MovieView} from "../movie-view/movie-view"
+import { useState, useEffect } from "react";
+import { MovieCard } from "../movie-card/movie-card";
+import { MovieView } from "../movie-view/movie-view";
+import { LoginView } from "../login-view/login-view";
+import  signupview  from "../signup-view/signup-view";
```

Comment:
This is the correct import path: `import { SingupView }  from "../signup-view/signup-view";`